[2024-10-12T18:36:05.656-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T18:36:05.671-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DesafioLH.export_final_output manual__2024-10-12T21:30:46.071524+00:00 [queued]>
[2024-10-12T18:36:05.682-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DesafioLH.export_final_output manual__2024-10-12T21:30:46.071524+00:00 [queued]>
[2024-10-12T18:36:05.682-0300] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-10-12T18:36:05.707-0300] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): export_final_output> on 2024-10-12 21:30:46.071524+00:00
[2024-10-12T18:36:05.717-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DesafioLH', 'export_final_output', 'manual__2024-10-12T21:30:46.071524+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/desafiolh.py', '--cfg-path', '/tmp/tmpi7tyibal']
[2024-10-12T18:36:05.716-0300] {standard_task_runner.py:72} INFO - Started process 95816 to run task
[2024-10-12T18:36:05.719-0300] {standard_task_runner.py:105} INFO - Job 91: Subtask export_final_output
[2024-10-12T18:36:05.808-0300] {task_command.py:467} INFO - Running <TaskInstance: DesafioLH.export_final_output manual__2024-10-12T21:30:46.071524+00:00 [running]> on host LAPTOP-9RLTGE5V.
[2024-10-12T18:36:05.954-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kaue.oliveira@indicium.tech' AIRFLOW_CTX_DAG_OWNER='Kaue Oliveira' AIRFLOW_CTX_DAG_ID='DesafioLH' AIRFLOW_CTX_TASK_ID='export_final_output' AIRFLOW_CTX_EXECUTION_DATE='2024-10-12T21:30:46.071524+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-12T21:30:46.071524+00:00'
[2024-10-12T18:36:05.961-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T18:36:05.997-0300] {python.py:240} INFO - Done. Returned value was: None
[2024-10-12T18:36:06.006-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T18:36:06.007-0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=DesafioLH, task_id=export_final_output, run_id=manual__2024-10-12T21:30:46.071524+00:00, execution_date=20241012T213046, start_date=20241012T213605, end_date=20241012T213606
[2024-10-12T18:36:06.061-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T18:36:06.075-0300] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-12T18:36:06.076-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
